requirements-dev.txt

```diff
@@ -1,5 +1,5 @@
-pytest
-pytest-cov
-pytest-django
+pytest==2.8.5
```

Comment:
Any reason we're pinning to an old version of pytest? Just making this like the others?